src/main/webapp/resources/js/components/samples/components/SequencingObjectList.tsx

```diff
@@ -198,11 +196,32 @@ export function SequencingObjectList({
     );
   };
 
+  /*
+  Check if the sequencing object should be automatically set as default
+   */
+  const checkSeqObjectAutoDefault = (type: string, index: number) => {
+    return (
```

Comment:
Simplified logic in 505268b1245ae1edc76877794c90c148867c1a94

Reply:
I think this can be simplified even further:

```typescript
return ((type === "pair") || (files.paired === undefined && type === "single") || (files.paired === undefined && files.singles === undefined && type === "fast5"));
```

Reply:
Simplified further in 98092a79e5c4f8c387818bc04b1c1757806e3e25